### DISCUSSION TITLE
files with many lines of JSON but some of them are in bad format. how to ignore those bad lines and continue to process the rest?

### DISCUSSION THREAD

**Discussion by bfzhao** [stedolan/jq#3011] [Q&A] [answered]
Given file with following content:
```
[]
{}
"hello"
12, 12
false
123
```

Note, the 4th line is in bad JSON format. jq just stops at this line after parsing error reported. I want jq to continue after emits a warning message.

**Comment by itchyny** [comment, marked as answer]
Use raw input option with `fromjson` filter.
```
jq -R 'fromjson? | [your query here]'
```
